modules/ags/src/main/scala/lucuma/ags/package.scala

```diff
@@ -151,15 +151,21 @@ extension (posAngleConstraint: PosAngleConstraint)
     vizTime:  Instant,
     duration: Duration
   ): Option[NonEmptyList[Angle]] =
-    TimeSpan
-      .fromDuration(duration)
-      .filter(_ > TimeSpan.Zero)
-      .flatMap: ts =>
-        posAngleConstraint match
-          case PosAngleConstraint.Fixed(a)               => NonEmptyList.of(a).some
-          case PosAngleConstraint.AllowFlip(a)           => NonEmptyList.of(a, a.flip).some
-          case PosAngleConstraint.ParallacticOverride(a) => NonEmptyList.of(a).some
-          case PosAngleConstraint.AverageParallactic     =>
-            averageParallacticAngle(site.place, tracking, vizTime, ts)
-              .map(a => NonEmptyList.of(a, a.flip))
-          case PosAngleConstraint.Unbounded              => UnconstrainedAngles
+    anglesToTestAt(
+      TimeSpan
```

Comment:
The vizTime is no longer validated unless we need to calculate the APA. I don't think this is a problem.

Reply:
We were making sure it was a valid TimeSpan and not Zero at the beginning of the method before. Now we only do it if we need it.

Reply:
It seems we validate `duration` but not `vizTime`